warehouse/events/models.py

```diff
@@ -130,28 +130,19 @@ def record_event(
         """Records an Event record on the associated model."""
         session = orm.object_session(self)
 
-        if request is not None:
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** ‏update the function signature to remove `= None` from the request kwarg, and thus no longer allow non-request-passing calls to `record_event`

Reply:
b683ea4ad